src/Adapters/Storage/Mongo/MongoSchemaCollection.js

```diff
@@ -151,18 +150,7 @@ class MongoSchemaCollection {
   // TODO: don't spend an extra query on finding the schema if the type we are trying to add isn't a GeoPoint.
   addFieldIfNotExists(className: string, fieldName: string, type: string) {
     return this._fechOneSchemaFrom_SCHEMA(className)
```

Comment:
Looking at this TODO. Is the this._fechOneSchemaFrom_SCHEMA() needed here? Would be nice if we didn't have to query.

Reply:
@flovilmart should I rebase? Also have you seen this comment.